ko/admin/config.rst

```diff
@@ -597,11 +597,11 @@ CUBRID 설치 시 생성되는 기본 데이터베이스 환경 설정 파일(**
     *   **$CUBRID/conf/hosts.conf** 파일의 형식은 **/etc/hosts** 와 동일하나 다음과 같은 몇 가지 제약이 있다.
 
         * **IPv4** 주소 만을 허용한다 (**IPv6** 는 허용하지 않는다).
-        * 아래와 같은 형식의 **alias** 는 허용하지 않는다. ::
+        * **alias** 는 허용하지 않는다. ::
 
            172.31.0.1 host1 alias1 alias2
 
-        * 아래와 같은 형식의 **alias** 는 허용한다. ::
+        * 같은 ip 주소가 여러 개 있는 경우 맨 윗 줄에 있는 {ip, hostname} 정보만 사용되며, 나머지는 무시된다. ::
 
            172.31.0.1 host1
            172.31.0.1 alias1
```

Comment:
Why don't we use another example instead of "alias1" because the user might be confused?
ex) 172.31.0.1 host1
      172.31.0.1 host2

Reply:
Did you mean:

AS-IS
172.31.0.1 host1
172.31.0.1 alias1

TO-BE
172.31.0.1 host1
172.31.0.1 host2

is this right?

Reply:
correct.

Reply:
It looks so good. I will apply it.

Thank you.

Reply:
good!

Reply:
I applied it. thank you!